lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -280,11 +332,103 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
         this.categoryControlVisible = false;
     }
 
-    cancelChanges() {
+    cancelChanges(buttonType: ButtonType, event: MouseEvent) {
+        event.stopPropagation();
+        this.toggleEditMode(buttonType);
         this.revertChanges();
         this.loadProperties(this.node);
     }
 
+    isEditingPanel(): boolean {
+        return (
+            (this.editable && this.hasMetadataChanged) ||
+            (this.editableTags && this.hasMetadataChanged) ||
+            (this.editableCategories && this.hasMetadataChanged) ||
+            (this.editableGroup?.editable && this.hasMetadataChanged)
+        );
+    }
+
+    showSnackbar(message: string): void {
+        this.snackBar.open(message, '', {
+            duration: 3000,
```

Comment:
Why do you override duration of snackbar? There is already global configuration for duration: 
<img width="900" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/9d0d7987-7852-4bc1-9740-d4f107b85a9b">

Reply:
Now I'm using existing snackbar design.